src/style.css

```diff
@@ -1,3 +1,79 @@
 body {
   margin: 0;
+  padding: 0;
+  background-color: rgb(203, 237, 241);
+  font-family: Roboto, Arial, sans-serif;
+
+  --text-color: #000;
+  --link-color: #00acdc;
+  --heder-hight: 60px;
+}
+
+header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  text-decoration: none;
+}
+
+.nav {
+  display: flex;
+  align-items: stretch;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  line-height: var(--heder-hight);
+  align-items: center;
+}
+
+.nav_item {
+  margin-left: 20px;
+}
+
+#linked_item {
+  margin-left: 0;
+}
+
+.nav_link {
+  color: var(--text-color);
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-decoration: none;
+  display: block;
+  cursor: pointer;
+}
+
+.nav_link:hover {
+  color: var(--link-color);
+}
+
+.is-active {
+  color: var(--link-color);
+  position: relative;
+}
+
+.is-active::after {
+  height: 4px;
+  width: 37px;
```

Comment:
something wrong with this implementation, because  you shouldn't give width here, please use another approach